perftracker/models/job.py

```diff
@@ -45,6 +45,7 @@ class JobModel(models.Model):
 
     project         = models.ForeignKey(ProjectModel, help_text="Job project", on_delete=models.CASCADE)
 
+    regression      = models.ForeignKey('perftracker.RegressionModel', help_text="Regression of this job", related_name="regr", null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
You forgot to add migrations (for addiging 'regression' key).

Also, it makes sense to write custom migration script where you link the jobs to existing Regressions...

Reply:
Understand